nekoyume/Assets/_Scripts/BlockChain/RPCAgent.cs

```diff
@@ -232,6 +261,7 @@ private void Awake()
             OnPreloadEnded
                 .Subscribe(_ => Analyzer.Instance.Track("Unity/RPC Preload Ended"))
                 .AddTo(_disposables);
+            Game.Event.OnUpdateAddresses.AddListener(UpdateSubscribeAddresses);
```

Comment:
Everytime `GetAvatarStates()` is called `UpdateSubscribeAddresses` seems to keep adding as a listener

Reply:
no, listener set in `Awake`